app/(admin)/openmailer/campaigns/[id]/SendPendingButton.tsx

```
@@ -64,7 +81,10 @@ export function SendPendingButton({
           throw err;
         }
       }
-      alert(`Pending emails sent. ${lastData.sent} of ${lastData.total} delivered.`);
+      const msg = lastData.quotaReached
+        ? `Daily quota reached. ${lastData.sent} of ${lastData.total} delivered. Use "Send pending" tomorrow or upgrade your Resend plan.`
+        : `Pending emails sent. ${lastData.sent} of ${lastData.total} delivered.`;
+      alert(msg);
```

ESLint

[error] Unexpected alert.

(no-alert)